scripts/bootstrap_history.py

```diff
@@ -134,7 +148,39 @@ def do_release(argv):
     release_file = _release_file(release_name + ".rst")
     release_info = TEMPLATE % release_name
     open(release_file, "w").write(release_info.encode("utf-8"))
-    print release_info
+    month = int(release_name.split(".")[1])
+    month_name = calendar.month_name[month]
+    year = release_name.split(".")[0]
```

Comment:
What about `year = '20' + release_name.split(".")[0]` instead of doing this in the template?